src/introduction.md

```diff
@@ -1,102 +1,98 @@
 # Introduction
 
-This book is the primary reference for the Rust programming language. It
-provides three kinds of material:
+This book is the primary reference for the Rust programming language.
+It provides three kinds of material:
 
   - Chapters that informally describe each language construct and their use.
-  - Chapters that informally describe the memory model, concurrency model,
-    runtime services, linkage model, and debugging facilities.
-  - Appendix chapters providing rationale and references to languages that
-    influenced the design.
+  - Chapters that informally describe the memory model, concurrency model, runtime services, linkage model, and debugging facilities.
+  - Appendix chapters providing rationale and references to languages that influenced the design.
 
 <div class="warning">
 
-Warning: This book is incomplete. Documenting everything takes a while. See
-the [GitHub issues] for what is not documented in this book.
+Warning:
+This book is incomplete. Documenting everything takes a while.
+See the [GitHub issues] for what is not documented in this book.
 
 </div>
 
+## Rust Releases
+
+Rust has a new language release every six weeks.
+The first stable release of the language was Rust 1.0.0, followed by Rust 1.1.0 and so on.
+Tools (`rustc`, `cargo`, etc.) and documentation ([Standard library], this book, etc.) are released with the language release.
+
+While tools often have the same version as the Rust language, the language exists as its own concept separately from them.
```

Comment:
I'm not entirely clear what this sentence is trying to say.  Can you elaborate?  I can kinda see that "Rust the Language" and "`rustc` the compiler" are two separate things that happen to be versioned together, but I'm uncertain if it needs to be called out. I think the previous sentence makes it clear how the tools are packaged and versioned.

Reply:
I wrote this before I wrote what was before it, and yeah, it's trying to say exactly that. It's important more so if there's another compiler that gets mainstream usage, but I can just remove it for now. The previous paragraph explains it clearly enough.